test/functional/breadcrumbs.ts

```diff
@@ -53,6 +60,43 @@ suite<Breadcrumbs>('gb-breadcrumbs', ({ flux, mount, stub, itMountsTag }) => {
 
       expect(model.queryCrumb).to.not.be.ok;
     });
+
+    describe('render with labels', () => {
+      it('should render with defaults', () => {
+        expect(model.labels[0].textContent).to.eq(DEFAULT_CONFIG.noResultsLabel);
+        expect(model.labels[1].textContent).to.eq(DEFAULT_CONFIG.correctedResultsLabel);
+
+        tag.updateQueryState(<any>{ originalQuery: ORIGINAL_QUERY });
+        expect(model.labels[0].textContent).to.eq(DEFAULT_CONFIG.resultsLabel);
+      });
+
+      it('should not render', () => {
+        tag._config.labels = false;
```

Comment:
should be `tag._config = { labels: false };` in case we end up modifying another test's dependencies